packages/cli/src/helper/util.ts

```diff
@@ -30,10 +30,16 @@ export interface Flags extends Options {
 export type Apicall<A extends unknown[], Return> = (...a: A) => Promise<Return>;
 
 export function showTable(data: object[], flags: Options) {
-  const columns = Object.getOwnPropertyNames(data?.[0]);
```

Comment:
hi @gadhagod ! awesome sleuthing. I think the issue is in this line - `data?.[0]` is returning `undefined` since there is no 0'th element, and then `Object.getOwnPropertyNames` is throwing an error since the argument is not an object as it expects, but `undefined`

I think a more elegant way to solve this would be update the argument here from `data?.[0]` to `data?.[0] ?? {}` (so that an empty object is used instead of `undefined` for empty results). What do you think of this approach instead?

Reply:
Thanks for the feedback @scottwmorris.
Agreed. I have pushed again to my fork, implementing the more beautiful solution to the problem you suggested.